W3CTRMANIFEST

```diff
@@ -0,0 +1,5 @@
+webrtc.html?shortName=webrtc respec
```

Comment:
The file is read by the CI script, but will also be read ultimately by the automatic publication system; I was hoping to document that in the comments, but it doesn't look like the format allows for it at the moment.